### DISCUSSION TITLE
Why the feature dimension in WikipediaNetwork dataset inconsistent with the origin paper?

### DISCUSSION THREAD

**Discussion by Xzh0u** [pyg-team/pytorch_geometric#3895] [Q&A]
# Questions:
- Why the feature dimension in WikipediaNetwork dataset inconsistent with the origin paper?
- What's the meaning of the feature provided by your WikipediaNetwork class?

# Descriptions:
I found that the feature dimension in WikipediaNetwork dataset is inconsistent with the origin paper.
The WikipediaNetwork dataset provided by WikipediaNetwork class in torch_geometric.datasets has the feature in the size of [2277, 128] (https://graphmining.ai/datasets/ptg/wiki/), which means each node has a [1, 128] feature. However, in the origin paper provided by your doc, the dimension of the feature should be far more larger as shown in the picture below. 
![image](https://user-images.githubusercontent.com/46866598/150295209-9c1b2af7-c15c-4b1b-b171-b03fb2ba08a0.png)

And your docs describes that node features represent several informative nouns in the Wikipedia pages. But the length of a dictionary should usually far more larger that 128. So is the node feature in the data you provided really means the nouns in the Wikipedia pages?

Thank you! Looking forward to your reply as soon as possible.

**Comment by rusty1s** [comment]
Duplicated discussion: https://github.com/pyg-team/pytorch_geometric/issues/3896
